src/store/breed-slice.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from "@reduxjs/toolkit";
```

Comment:
Why are you using this libary? what were you trying to do?

Reply:
Redux Toolkit can help you make your Redux code better. Advanced of Redux